Mistica/Source/Components/Callout/Callout.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  Callout.swift
+//
+//  Made with ❤️ by Novum
+//
+//  Copyright © Telefonica. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+private enum Constants {
+    static let cornerRadius = CGFloat(4)
```

Comment:
I think we prefer this form:

```suggestion
    static let cornerRadius: CGFloat = 4
```